src/Symfony/Component/Console/Event/ConsoleErrorEvent.php

```diff
@@ -67,46 +58,26 @@ public function setError($error)
     }
 
     /**
-     * Marks the error/exception as handled.
-     *
-     * If it is not marked as handled, the error/exception will be displayed in
-     * the command output.
-     */
-    public function markErrorAsHandled()
-    {
-        $this->handled = true;
-    }
-
-    /**
-     * Whether the error/exception is handled by a listener or not.
-     *
-     * If it is not yet handled, the error/exception will be displayed in the
-     * command output.
+     * Sets the exit code.
      *
-     * @return bool
-     */
-    public function isErrorHandled()
-    {
-        return $this->handled;
-    }
-
-    /**
-     * @deprecated Since version 3.3, to be removed in 4.0. Use getError() instead
+     * @param int $exitCode The command exit code
      */
-    public function getException()
+    public function setExitCode($exitCode)
     {
-        @trigger_error(sprintf('The %s() method is deprecated since version 3.3 and will be removed in 4.0. Use ConsoleErrorEvent::getError() instead.', __METHOD__), E_USER_DEPRECATED);
+        $this->exitCode = (int) $exitCode;
 
-        return parent::getException();
+        $r = new \ReflectionProperty($this->error, 'code');
+        $r->setAccessible(true);
+        $r->setValue($this->error, $this->exitCode);
```

Comment:
because the other part of the exception handling uses getCode as the way to set the exit status code from an exception